python/tools/wlr-libpy/src/bld_cfg.rs

```diff
@@ -15,20 +15,32 @@ const LIBPYTHON_URL: &str = "https://github.com/vmware-labs/webassembly-language
 pub fn configure_static_libs() -> Result<LibsConfig, BoxedError> {
     let mut libs_config = LibsConfig::new();
 
-    let wasi_deps_path = Path::new(WASI_DEPS_PATH);
+    let out_dir = if let Ok(mut out_dir) = std::env::var("OUT_DIR") {
+        if let Some(position) = out_dir.rfind("/wasm32-wasi/") {
```

Comment:
I think we should convert the OUT_DIR to a `PathBuf`, and use `std::path::Path::components` to find out. This will be more solid if the trailing slash is missing.